erna/scripts/rename_columns.py

```diff
@@ -0,0 +1,64 @@
+import click
+import numpy as np
+import h5py
+from fact.instrument import camera_distance_mm_to_deg
+import logging
+import pandas as pd
+
+from ..hdf_utils import rename_columns, theta_columns, theta_deg_columns
+
+
+@click.command()
+@click.argument('inputfile', type=click.Path(exists=True, dir_okay=False))
+@click.option('-k', '--key', help='HDF5 key in inputfile', default='events')
+@click.option('-f', '--fmt', type=click.Choice(['h5py', 'pandas']), help='Type of hdf file', default='h5py')
+def main(inputfile, key, fmt):
+    '''
+    Rename columns in hdf5 file to the new format, add theta columns in degrees
+    e.g.
+    COGx -> cog_x,
+    Theta_Off_1 -> theta_off_1,
+    MCORSIKA_EVT_HEADER.fTotalEnergy -> corsika_evt_header_total_energy
+
+    INPUTFILE: A pandas-style hdf5 file
```

Comment:
is h5py a pandas-style hdf5 file? Or does this need comment need an update?

Reply:
If I understand correctly it could be both. Maybe put the `pandas-style` in brackets